#88[feature] Controller 테스트 코드 작성

goodisgun opened the pull request:
(PostController, CommentController, ReCommentController, LikeController)

goodisgun committed:
#88[feature] Controller 테스트 코드 작성
 (PostController, CommentController, ReCommentController, LikeController)

Dripmaster44 reviewed (dismissed):
확인 완료했습니다

ahnhadi commented on src/main/java/com/petfam/petfam/dto/post/PostResponseDto.java:
```diff
@@ -5,9 +5,12 @@
+import lombok.Setter;
 
 @Getter
+@Setter
```
테스트코드때문에 setter를 넣으신건가요?

goodisgun replied:
네 지금 수정하려고 하고있습니다 . 우선 테코가 안돌아가서 Controller리팩토링하면서 재적용해보겠습니다!

goodisgun committed:
Merge branch 'dev' into feature/#88

goodisgun committed:
#88[refactor] Controller Refactoring
 (PostController, LikeController
  => ResponseEntity 형태로 통일

goodisgun committed:
Merge remote-tracking branch 'origin/feature/#88' into feature/#88

ahnhadi commented on src/main/java/com/petfam/petfam/controller/PostController.java:
```diff
@@ -73,22 +73,24 @@ public ResponseEntity<Page<AllPostResponseDto>> getPosts(@RequestParam(defaultVa
 
-  @PatchMapping("/{id}")
-  public String updatePost(@PathVariable Long id,
+  @PostMapping("/{id}")
```
이거 Update라서 PatchMapping 쓰셔야 하는데 Post로 쓰신 이유가 있으신가요?

goodisgun replied:
아.. 제가 잘못 수정한거 같습니다. 리팩토링때 수정해서 반영하겠습니다!

goodisgun committed:
#88[refactor] Controller Refactoring
 - Comment, Post, User Setter제거
 - PostController 오탈자 수정 PostMapping => PatchMapping

goodisgun committed:
Merge branch 'dev' into feature/#88

ahnhadi reviewed (approved):
수정하신사항 다 확인했습니다
요청드린사항 수정 다 해주셨네요
고생하셨습니다!
